src/Microsoft.Identity.Client/Config.cd

```diff
@@ -0,0 +1,136 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I synced your branch but I can't see this file. Strange...

Reply:
Anyway, I'm more of an code map / dgml person myself, but I'm biased :)

Reply:
i actually don't know where this came from and why it's here.  will delete.

---
In reply to: [241519250](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/723#discussion_r241519250) [](ancestors = 241519250)

Reply:
This is from JM and it's in the file system but it's not in the project.  We can look at redoing this as codemap but for now i think it's helpful.

---
In reply to: [241519250](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/723#discussion_r241519250) [](ancestors = 241519250)